medicover_session.py

```diff
@@ -95,7 +102,7 @@ def log_in(self):
         # https://oauth.medicover.pl/external?provider=IS3&signin=944f8051df4165a710e592dd7f8a&owner=Mcov_Mol&ui_locales=pl-PL
 
         response = self.session.get(
-            "https://oauth.medicover.pl/external",
+            "%s/external" % BASE_OAUTH_URL,
```

Comment:
Minor remark: let's use f-string instead of %-formatting. Just to keep the code more consistent :wink: